src/ol/geom/geometry.js

```diff
@@ -170,6 +170,16 @@ ol.geom.Geometry.prototype.getExtent = function(opt_extent) {
 
 
 /**
+ * Rotate the geometry around a given coordinate. This modifies the geometry
+ * coordinates in place.
+ * @param {number} angle Rotation angle in radians.
```

Comment:
Always nice to specify the direction (e.g. "Rotation angle in radians (counter-clockwise)").